File: result/translate/starchat/python2java/java/COUNT_PAIRS_TWO_SORTED_ARRAYS_WHOSE_SUM_EQUAL_GIVEN_VALUE_X_1.java
public static int f_gold ( int arr1 [ ], int arr2 [ ], int m, int n, int x ) {
        Set < Integer > us = new HashSet < > ( );
        for ( int i = 0 ; i < m ; i ++ ) us.add ( arr1 [ i ] );
        int count = 0;
        for ( int j = 0 ; j < n ; j ++ ) if ( x - arr2 [ j ] in us ) count ++;
        return count;
    }

File: result/translate/transcoder/java2cpp/c++/COUNT_PAIRS_TWO_SORTED_ARRAYS_WHOSE_SUM_EQUAL_GIVEN_VALUE_X_1.cpp
int f_gold ( int arr1 [ ] , int arr2 [ ] , int m , int n , int x ) {
  int count = 0 ;
  set < int > us ;
  for ( int i = 0 ;
  i < m ;
  i ++ ) {
    us . insert ( arr1 [ i ] ) ;
  }
  for ( int j = 0 ;
  j < n ;
  j ++ ) if ( us . count ( x - arr2 [ j ] ) ) count ++ ;
  return count ;
}